c7n/resources/rds.py

```diff
@@ -130,58 +129,10 @@ def generate_arn(self):
                 account_id=self.account_id, resource_type='db', separator=':')
         return self._generate_arn
 
-    def get_source(self, source_type):
-        if source_type == 'describe':
-            return DescribeRDS(self)
-        elif source_type == 'config':
-            return ConfigRDS(self)
-        raise ValueError("Unsupported source: %s for %s" % (
-            source_type, self.resource_type.config_type))
-
 
-class DescribeRDS(DescribeSource):
-
-    def augment(self, dbs):
-        filter(None, _rds_tags(
-            self.manager.get_model(),
-            dbs,
-            self.manager.session_factory,
-            self.manager.executor_factory,
-            self.manager.generate_arn,
-            self.manager.retry))
-        return dbs
-
-
-class ConfigRDS(ConfigSource):
```

Comment:
umm.. this stuff shouldn't be removed, you want the universal tag augment in the DescribeRDS source, the ConfigRDS stays as is, please put back get_source, DescribeRDS, and ConfigRDS. you can yank _rds_tags.